.github/actions/repo-gardening/src/tasks/clean-labels/index.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Internal dependencies
+ */
+const debug = require( '../../debug' );
+const getLabels = require( '../../get-labels' );
+
+/* global GitHub, WebhookPayloadPullRequest */
+
+/**
+ * Manage labels when a PR gets merged.
+ *
+ * @param {WebhookPayloadPullRequest} payload - Pull Request event payload.
+ * @param {GitHub}                    octokit - Initialized Octokit REST client.
+ */
+async function cleanLabels( payload, octokit ) {
+	const { pull_request, repository, action } = payload;
+	const { number } = pull_request;
+	const { name: repo, owner } = repository;
+	const ownerLogin = owner.login;
+
+	// Normally this only gets triggered when PRs get closed, but let's be sure.
+	if ( action !== 'closed' ) {
+		debug( `clean-labels: PR #${ number } is not closed. Aborting.` );
+		return;
+	}
+
+	// Get array of all labels on the PR.
+	const labelsOnPr = await getLabels( octokit, ownerLogin, repo, number );
+
+	// List of all labels we want to remove.
+	const labelsToRemove = [
+		'[Status] Ready to Merge',
+		'[Status] Needs Review',
+		'[Status] Needs Team Review',
+		'[Status] In Progress',
+		'[Status] Needs Author Reply',
+		'[Status] Needs Design Review',
+		'[Status] Needs i18n Review',
+		'[Status] String Freeze',
+	];
+
+	const labelsToRemoveFromPr = labelsOnPr.filter( label => labelsToRemove.includes( label ) );
+
+	if ( ! labelsToRemoveFromPr.length ) {
+		debug( `clean-labels: no labels to remove from #${ number }. Aborting.` );
+	}
```

Comment:
```suggestion
	return;
	}
```

Should there be a return here?